trillian_log_api.proto

```diff
@@ -240,7 +261,7 @@ service TrillianLog {
       };
     }
 
-    // Corresponds to the LeafReader API
+    // DO NOT USE - FOR DEBUGGING ONLY (FIXME: remove or move to a debugging interface)
```

Comment:
This is for integration tests not debugging really. It requires a full scan of distributed storage and is very slow.